x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -210,55 +211,67 @@ function UserSettingsEditor({
       <FormRow
         name="data.userSettings.darkMode"
         label={
-          <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
-            <EuiFlexItem grow={false}>
-              <FormLabel for="data.userSettings.darkMode">
-                <FormattedMessage
-                  id="xpack.security.accountManagement.userProfile.userSettings.theme"
-                  defaultMessage="Mode"
-                />
-              </FormLabel>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>{renderHelpText(isThemeOverridden)}</EuiFlexItem>
-          </EuiFlexGroup>
+          <FormLabel for="data.userSettings.darkMode">
+            <FormattedMessage
+              id="xpack.security.accountManagement.userProfile.userSettings.theme"
+              defaultMessage="Mode"
+            />
+          </FormLabel>
         }
         fullWidth
       >
-        <EuiKeyPadMenu
-          aria-label={i18n.translate(
-            'xpack.security.accountManagement.userProfile.userSettings.themeGroupDescription',
-            {
-              defaultMessage: 'Elastic theme',
-            }
-          )}
-          data-test-subj="themeMenu"
-          checkable={true}
+        <EuiToolTip
+          data-test-subj="themeOverrideTooltip"
+          content={
+            isThemeOverridden ? (
+              <FormattedMessage
+                id="xpack.security.accountManagement.userProfile.overriddenMessage"
+                defaultMessage="This setting is overridden by the Kibana server and can not be changed."
+              />
+            ) : (
+              ''
+            )
+          }
         >
```

Comment:
@MichaelMarcialis That was the approach I was originally aiming for, but because EuiToolTip is meant to wrap another component (in this case, one that is always rendered), it was complicated. Do you have an example of somewhere this is already being done?